client/src/components/game-state-display/game-area/GameArea.js

```diff
@@ -4,25 +4,99 @@ import FadeOutText from './anim/FadeOutText';
 import ReactCountdownClock from 'react-countdown-clock';
 import './game-area.css';
 
-const GameArea = ({ socket, liveMessage, roundDuration, canvasOptions }) => {
-  return (
-    <Fragment>
-      <Canvas io={socket} canvasOptions={canvasOptions}/>
-      <div className="live-message">
-        <FadeOutText text={liveMessage}></FadeOutText>
-      </div>
-      <div className="timer">
-        <ReactCountdownClock
-          className="timer"
-          seconds={roundDuration}
-          color="#D28152"
-          weight={5}
-          alpha={0.9}
-          size={48}
-        />{' '}
-      </div>
-    </Fragment>
-  );
-};
+class GameArea extends React.Component {
```

Comment:
@existentialcoder : any particular reason why you went for class components instead of function components ?

Reply:
Just a personal preference. Have been writing class components for a quite some time now. Also fan of life cycle methods